resources/js/features/messages/components/ReadableMessageCard/ReadableMessageCard.tsx

```diff
@@ -0,0 +1,35 @@
+import type { FC } from 'react';
+
+import { ShortcodeRenderer } from '@/common/components/ShortcodeRenderer';
+import { UserAvatar } from '@/common/components/UserAvatar';
+import { usePageProps } from '@/common/hooks/usePageProps';
+import { formatDate } from '@/common/utils/l10n/formatDate';
+
+interface ReadableMessageCardProps {
+  message: App.Community.Data.Message;
+}
+
+export const ReadableMessageCard: FC<ReadableMessageCardProps> = ({ message }) => {
+  const { auth } = usePageProps();
+
+  return (
+    <div className="rounded bg-embed px-2.5 py-1.5">
+      <div className="flex items-center gap-2">
+        <UserAvatar {...message.author!} size={24} />
+
+        <p className="smalldate">
+          {formatDate(
+            message.createdAt,
+            auth?.user.preferences.prefersAbsoluteDates ? 'MMM DD, YYYY, HH:mm' : 'LLL',
+          )}
+        </p>
+      </div>
+
+      <hr className="my-2 w-full border-embed-highlight" />
+
+      <div style={{ wordBreak: 'break-word' }}>
+        <ShortcodeRenderer body={message.body} />
```

Comment:
`<br>`s are being escaped.

Before:
![image](https://github.com/user-attachments/assets/db6b42df-e9dc-4eda-985e-6b733a7d6004)

After:
![image](https://github.com/user-attachments/assets/ff2c8e1f-da00-4f6e-9e05-78b80c50fbde)

Reply:
Hmm... looks like this issue is unique to messages and stopped happening sometime around 2021.

Before I start fishing for a solution in the React code, I'm thinking it may actually be better to resolve this via a command to update the database. Would you have any reservations against such a command?

Reply:
No. As long as the content is visually correct, the way its stored can be sanitized.

Reply:
I think the current "Before" is visually incorrect:
```
Line One



Line Two
```

This should probably be:
```
Line One

Line Two
```
I'll write a command to convert those break tags over.

Reply:
I actually think this SQL query will resolve the issue without the need for a command:
```sql
UPDATE messages 
SET body = TRIM(
    REPLACE(
        REPLACE(
            REPLACE(body, '<br />\n', ''),
            '<br />', ''
        ),
        '\n\n', '\n'
    )
)
WHERE body LIKE '%<br />%';
```
If this is agreeable, I'll add it to the PR description & attach the appropriate label.

Reply:
The innermost replace is unnecessary (and undesirable). There are no matches because the database has Windows newlines (`\r\n`). If there were matches, you'd be removing newlines where you shouldn't. Each of these lines ends with a `<br />\r\n`. If you stripped the `<br />` and the newline, these lines would all run together:
```
Achievement: [ach=10948]<br />
Game: [game=1505]<br />
Problem: Doesn't trigger<br />
Comment: <br />
```

Reply:
There are quite a few rows with `\n\n`. And I don't think that collapsing them is desirable (last replace):
```
select id, body, replace(body, '\n\n', '\n') from messages where body like '%\n\n%' limit 1\G;
```
```
...
Emulator: RetroArch (Snes9x)

This ticket will be raised and will be available for all developers to inspect and manage at the following URL:
https://retroachievements.org/ticketmanager.php?i=32524

Thanks!
```
becomes
```
Emulator: RetroArch (Snes9x)
This ticket will be raised and will be available for all developers to inspect and manage at the following URL:
https://retroachievements.org/ticketmanager.php?i=32524
Thanks!
```

Reply:
Thanks for those examples to trial & error against. Here's a revised version of the query that I think should behave quite a bit better:
```sql
UPDATE messages 
SET body = TRIM(
    REPLACE(
        REPLACE(body, '<br />\r\n', '\r\n'),
        '<br />', '')
)
WHERE body LIKE '%<br />%';
```

Reply:
Isn't the inner replace redundant now?

Reply:
Yup 🥴 

This is more concise:
```sql
UPDATE messages 
SET body = TRIM(REPLACE(body, '<br />\r\n', '\r\n'))
WHERE body LIKE '%<br />%';
```

Reply:
I meant:
```
UPDATE messages 
SET body = TRIM(REPLACE(body, '<br />', ''))
WHERE body LIKE '%<br />%';
```

Reply:
That query does also seem to work with my testing.

Reply:
Replacing "Abc" with "bc" is just "remove A if next to bc". Your previous queries were removing all A's.

Reply:
Updated the PR description with this SQL query. Not sure if there's any other feedback currently that I need to work through.